scripts/test/lint.py

```diff
@@ -374,6 +377,7 @@ class UsesCategoryFromNuspec(Lint):
         "x64dbgpy.vm",
         "vscode.extension.",
         "chrome.extensions.vm",
+        "dcode.vm",
```

Comment:
[nitpick]  I think the comment should be in the same or previous line as the dcode exclusion. Otherwise I think it is likely that we forget to remove the comment when we remove the dcode exclusion. Please use the URL instead of the issue number so that it is clear what issue is meant and how to access it. 
```suggestion
        # exclude dcode as it fails to install: https://github.com/mandiant/VM-Packages/issues/1176
        "dcode.vm",
```